_pages/about.md

```diff
@@ -12,32 +12,24 @@ toc_sticky: true
 toc_icon: "align-justify"
 ---
 
-# Who we are
-
-We are Bjoern and Kirthihan, two enthusiastic computer engineers with a passion
-for embedded systems, digital design and efficient software programming. Our
-journey together began when we studied electrical engineering at RWTH Aachen
-University in Germany. At the Institute for Communication Technologies and
-Embedded Systems, we worked as student assistants, where we contributed to
-different projects in the fields of hardware accelerators for machine learning,
-FPGA design, processor prototyping and embedded software. Nowadays, Bjoern works
-as a firmware developer in the audio industry. Kirthihan works as a digital
-hardware developer in the cybersecurity industry.
-
 # What you can expect
```

Comment:
If this is the new first section title, it doesn't fit well anymore, IMO. Either we could leave it out, or use a different title.

Reply:
What would you prefer? Something like "Bits&Volts - A blog for everyone"?

Reply:
Sounds good.